spec/latest/json-ld-api/index.html

```diff
@@ -1620,8 +1624,8 @@ <h3>Algorithm</h3>
                           and (<code>@language</code>-lowercased
                           <em>language</em>),
                           <span class="changed">unless <em>item</em> is <code>null</code></span>.
-                          <span class="ednote">From <a href="#issue-480">Issue 480</a>
-                          the <a>language tag</a> <code>@none</code> represents no language.</span></li>
+                          <span class="changed">If <em>language</em> is <code>@none</code>,
+                            or expands to <code>@none</code>, do not set the <code>@language</code> member.</span>
```

Comment:
Do we need explicit mention that the underlying RDF datatype is different? (xsd:string versus rdf:langString)

Reply:
Right, I agree that @none is json-ld flavored sugar... but the RDF Concepts spec asserts different datatypes based on the presence or lack of a language tag. At least that's my understanding of https://www.w3.org/TR/rdf11-concepts/#section-Graph-Literal

If there's a language tag, then the data type is necessarily rdf:langString. And if there isn't, then it's not. If there isn't another data type, it's an xsd:string.  

Which we can be completely silent about, but I'd like us to have thought about that silence :)

Reply:
[Language maps](https://json-ld.org/spec/latest/json-ld/#language-maps) have a proscribed form, where the key is a string, and the value is either a string or an array of strings, so the expansion of a map value of `@none`, would be like `{"@value": "non-languaged-tagged value"}`, thus no datatype or language. The [Object to RDF Conversion](https://json-ld.org/spec/latest/json-ld-api/#object-to-rdf-conversion) algorithm renders this as a literal without datatype or language, which, as you note, is semantically equivalent to having the datatype `xsd:string`.

Do you think this needs some exposition in either document?